pax-cli/src/main.rs

```diff
@@ -207,7 +204,7 @@ fn perform_nominal_action(
                     let target = args.value_of("target").unwrap().to_lowercase();
                     let path = args.value_of("path").unwrap().to_string(); //default value "."
 
-                    let working_path = Path::new(&path).join(".pax");
+                    let working_path = std::env::temp_dir().join(".pax");
```

Comment:
Puts `.pax` into the temporary directory of each platform. Please feel free to let me know if this change is not desired.